### PR TITLE
Bugfix/dapp broken transfer steps on small screens

### LINKED ISSUE #2768: PFS options screen while doing mediated transfers is distorted
## Steps to Reproduce
1. Perform mediated transnfer
2. 

## Expected Result
- PFS options should be displayed correctly

## Actual Result
- 
pfs_options_transfer_distorted

### PR DESCRIPTION
**Pull request by weilbith**
**Thank you for submitting this pull request :)**

Fixes #2768 and [this comment](https://github.com/raiden-network/light-client/issues/2768) of #2604

**Short description**
Sorry, the commits are not so clean. It was a nightmare to test this...

**Definition of Done**

- [x] Steps to manually test the change have been documented
- [x] Acceptance criteria are met
- [ ] Change has been manually tested by the reviewer (dApp) 

**Steps to manually test the change (dApp)**

1. Try to do a mediated transfer with multiple routes on `320x568` screen size
2. Make everything looks good.

### CODE REVIEW

File: raiden-dapp/src/components/stepper/StepperDivider.vue [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      'stepper-divider--skipped': skipped,
+    }"
+  />
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from 'vue-property-decorator';
+
+@Component
+export default class StepperStep extends Vue {
```
Comment (taleldayekh) [inline raiden-dapp/src/components/stepper/StepperDivider.vue:9]: `class StepperDivider extends...` not `StepperStep`

File: raiden-dapp/tests/unit/views/transfer-steps-route.spec.ts [hunk 2]
```diff
@@ -97,6 +97,7 @@ async function createWrapper(
   const wrapper = shallowMount(TransferSteps, {
     store,
     vuetify,
+    // stubs: { 'action-button': ActionButton },
```
Comment (taleldayekh) [inline raiden-dapp/tests/unit/views/transfer-steps-route.spec.ts:100]: Forgotten comment?

**Review by taleldayekh — APPROVED** [review]
Looks good codewise, thank you. Trying to make a mediated transfer to see the actual UI.

File: raiden-dapp/src/components/stepper/StepperDivider.vue (hunk 1 above)
Comment (weilbith) [reply raiden-dapp/src/components/stepper/StepperDivider.vue]: ✔️

File: raiden-dapp/tests/unit/views/transfer-steps-route.spec.ts (hunk 2 above)
Comment (weilbith) [reply raiden-dapp/tests/unit/views/transfer-steps-route.spec.ts]: ✔️

**Comment by taleldayekh** [comment]
Tested the UI now and it looks good, you can go ahead and merge this now.

**Pull request merged**